doc/content/lfr/cardinal_7pincell/Cardinal_7pin_LFR_demo.md

```diff
@@ -0,0 +1,438 @@
+allowed to converge first (it is not highly dependent on temperature), which then drives the CFD calculation. (B) is more computationally efficient for weakly coupled problems and also ensures convergence of the temperature distribution.
+
+### Data transfer style=font-size:125%
+
+[transfer] shows the data transfer scheme between different physics solvers. For transferring volumetric heat source from Griffin to H.C., Griffin computes constant volumetric heat source per element and the value at the centroid of an element in the H.C. mesh is queried from the finite element solution in the Griffin mesh and used as a constant value over the element (see +MultiAppShapeEvaluationTransfer+ [!citep](Lindsay2022)).
+The transferred heat source is normalized to preserve the original total power using MOOSE’s conservative transfer features. 
+For transferring the temperature field from H.C. to Griffin for Doppler and coolant density feedback, temperature values at the three closest nodes in the H.C. mesh to the target node in the Griffin mesh are
+interpolated using inverse distance weighting (see +MultiAppGeometricInterpolationTransfer+ [!citep](Lindsay2022)).
+Nearest points are searched only in the solid region for solid temperatures and in the fluid region for fluid temperatures. 
+Even though H.C. does not need the fluid domain for solving the heat
+conduction equation, the H.C. problem contains a mesh for the fluid domain in order to receive the coolant temperature field transferred from Cardinal (on its way to Griffin) and compute its L2-norm for convergence check. 
+Note that Griffin does not need a fine mesh in the coolant region for
+neutronics solution, and a fine coolant mesh in H.C. does not degrade computational efficiency except in memory-limited cases.
+
+!media lfr/cardinal_7pin/transfer.png
+       style=width:100%
+       id=transfer
+       caption=Data transfer scheme among Griffin, H.C., and Cardinal.
+
+For the conjugate heat transfer, H.C. computes constant monomial heat flux per element at the cladding outer surfaces and the duct inner surface. 
+Note that the heat flux variable computed by H.C. is an elemental type, while the variable automatically added by Cardinal on the mirror mesh
+of the NekRS native mesh, is the nodal type. 
+Thus, the heat flux value at the nearest centroid of an element in the H.C. mesh is taken to the target node in the Cardinal mesh using
++MultiAppGeneralFieldNearestNodeTransfer+ [!citep](Lindsay2022). 
+If there are multiple nearest centroids near parallel process boundaries, parallel resolution between nearly equi-distant points may be inaccurate unless extense searches are done over every process. 
+To avoid this, the +greedy_search+ option was turned on to check the points of all the processors. This transfer takes place separately for different interfaces (e.g. duct-coolant and clad-coolant) with spatial restrictions on source and target boundaries for the search. 
```

Comment:
Likely so, but not certain. If this is a concern, I can connect you to the model developers for further discussion. Otherwise, I tend to leave it as it is.

Reply:
There were histories to this (many bug fixes etc.) but even with the bug fix we saw transferred data were messed up using multiple processors while they were OK using a single processor. This was discussed when we (ANL and INL) had a TEAMS meeting and the possible reason for this is the following sentence.
`If there are multiple nearest centroids near parallel process boundaries, parallel resolution between nearly equi-distant points may be inaccurate unless extense searches are done over every process.`
So we fixed this by using `greedy_search` even though it lowers performance. Performance degradation was not an issue at all since the time taken for data transfer was negligible compared to physics simulations.
@yukkiezy